app/middleware/__init__.py

```
@@ -0,0 +1,28 @@
+    setup_exception_handlers,
+    ErrorResponse,
+)
+
+__all__ = [
+    "CorrelationIdMiddleware",
+    "LoggingMiddleware",
+    "ErrorHandlerMiddleware",
+    "setup_exception_handlers",
+    "ErrorResponse",
+    "get_correlation_id",
+]
```

Ruff (0.11.9)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)